Source/WebCore/editing/Editor.cpp

```diff
@@ -2894,7 +2894,9 @@ void Editor::markAndReplaceFor(const SpellCheckRequest& request, const Vector<Te
 
     for (unsigned i = 0; i < results.size(); i++) {
         auto spellingRangeEndOffset = paragraph.checkingEnd() + offsetDueToReplacement;
-        TextCheckingType resultType = results[i].type;
+        if (!results[i].type.hasExactlyOneBitSet())
+            continue;
```

Comment:
Probably add an ASSERT_NOT_REACHED here.  If regular code hits this, we want an engineer to notice it with an assertion.